theano/gpuarray/corr3d_gemm.c

```diff
@@ -201,6 +207,7 @@ KERNEL void dilated_col2im3d_kernel(const ga_size n,
 }
 
 #kernel col2im3d_kernel : size, *, size, size, size, size, size, size, size, size, size, size, size, size, size, size, size, size, size, *, size, size :
+
```

Comment:
just for consistency with the rest of the code. But I agree, it look clearner. So i'm good to keep it. But we should ideally do this everywhere at some point. I'm merging.